main.py

```diff
@@ -1,10 +1,55 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def set_skills(race: Race, race_skills: list[dict]) -> None:
+    for skill in race_skills:
+        if not Skill.objects.filter(name=skill["name"]).exists():
+            Skill.objects.create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+
+def set_guild(guild: dict) -> None:
+    if not Guild.objects.filter(name=guild["name"]).exists():
+        Guild.objects.create(
+            name=guild["name"],
+            description=guild["description"]
+        )
```

Comment:
try to use get_or_create method to simplify ur code )
also, this method could return an object of guild to remove 44 line

Reply:
I was thinking about it, but decided to use advised method exists(). I will fix it